hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/metrics/JRaftMetrics.java

```diff
@@ -172,35 +168,33 @@ private static void registerHistogram(String group, String name,
 
         String baseName = PREFIX + "." + name.toLowerCase();
 
-        HistogramWrapper wrapper = toWrapper(histogram);
-
-        Gauge.builder(baseName + ".median", wrapper, (d) -> d.getSnapshot().getMedian())
+        Gauge.builder(baseName + ".median", histogram, (d) -> d.getSnapshot().getMedian())
```

Comment:
Will getSnapshot() be called frequently?

Reply:
yes, request pull metrics from Prometheus every time

Reply:
It turns out that the logic is to only request a snapshot after > 30 seconds, 
which will cause the indicator to be empty. Therefore,
 it is requested once every time the indicator is pulled. 
The frequency is not high. 

Online, it is usually pulled once every 15 seconds.

Reply:
ok